containers/utils/get-deb-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
```

Comment:
As the original author of an earlier version of this script (see https://gitlab.com/swinslow/contcompl/-/blob/master/take2/get-sources.sh) -- if you would like to mark this as the following instead, that would be fine with me:

```
SPDX-License-Identifier: EPL-2.0 OR MIT
```

Just offering that as an option, for better alignment with Zowe's own license  :)